snooty/parser.py

```diff
@@ -613,6 +621,29 @@ def __init__(
         self.asset_dg: "networkx.DiGraph[FileId]" = networkx.DiGraph()
         self._expensive_operation_cache = Cache()
 
+        published_branches: Optional[PublishedBranches] = self.get_parsed_branches()
+        if published_branches:
+            self.published_branches: PublishedBranches = published_branches
```

Comment:
We never actually use this anywhere else. Does it need to be an instance property?

Reply:
It's used in a test [here](https://github.com/mongodb/snooty-parser/pull/42/files#diff-63fe176602884ead559d986042bccc4eR115-R118)! Do you think it's worth keeping for this?

Reply:
That test can use the `on_published_branches` handler I think!